### PR TITLE
Added new carousel component

### PR DESCRIPTION
**Pull request by skmadridista**
Added a responsive Carousel Component.
image
Fix for #11

### CODE REVIEW

File: src/components/CarouselComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            </Ratio>
+          ) : (
+            <img
+              className="d-block mx-auto rounded"
+              src={image.src}
+              alt={image.alt}
+              width="800" // Set your preferred width for larger screens
+              height="400" // Set your preferred height for larger screen
+            />
+          )}
```
Comment (Saif-Azam-Khan) [inline src/components/CarouselComponent.jsx:47]: Create a new Image component with the same logic and use it here. 
The new component can be used for any future requirements.

Comment (skmadridista) [reply src/components/CarouselComponent.jsx:47]: Sure, I am on it...!

Comment (skmadridista) [reply src/components/CarouselComponent.jsx]: @Saif-Azam-Khan I have pushed the requested changes, please review.

File: src/static/css/CarouselComponent.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  
+  /* Media query for screens with a maximum width of 768px (typical for mobile devices) */
+  @media (max-width: 768px) {
+    .carousel {
+      max-width: 100%; /* Full width for smaller screens */
+      height: auto; /* Allow the height to adjust proportionally */
+    }
+  }
+
+
```
Comment (Saif-Azam-Khan) [inline src/static/css/CarouselComponent.css:15]: Please remove the additional line break from this file

**Review by Saif-Azam-Khan — APPROVED** [review]
LGTM

**Pull request merged**